src/main/res/values/strings.xml

```diff
@@ -14,6 +14,10 @@
     <string name="nav_drawer_action">Compass</string>
     <string name="main_tab_title">My Journey</string>
 
+    <!-- Time/Recurrence pickers -->
+    <string name="time_picker_confirmation_toast">You have selected </string>
+    <string name="recurrence_picker_confirmation_toast">Reminder Saved!</string>
```

Comment:
Use - "You have selected %1$s", for which you can use getString(int id, String args) as in 
```
getString(R.string.time_picker_confirmation_toast, mTime)
```